scripts/compare.py

```diff
@@ -0,0 +1,232 @@
+					continue
+
+				# Get overlap
+				overlap_polygons = new_polygon.intersection(old_polygon)
+
+				# Ensure this is a multipolygon
+				if not isinstance(overlap_polygons, shapely.geometry.MultiPolygon):
+					overlap_polygons = shapely.geometry.MultiPolygon([overlap_polygons])
+
+				# Check each overlap separately
+				for overlap_polygon in overlap_polygons:
+					export_overlap = False
+
+					if overlap_polygon.area > 0:
+						if overlap_polygon.area <= (stroke_width*stroke_width * 2.0):
+							#FIXME: Check if this is in a pad or via; if not in pad or via: error
+							print("note: Small overlap, assuming connection")
+							touching[new_element_id] = old_element_id
+						else:
+							print("error: Large overlap")
+							conflict[new_element_id] = old_element_id
+							export_overlap = True
+
+					if export_overlap:
+						overlaps[old_element_id] = overlaps.get(old_element_id, []).append(overlap_polygon)
+
+		# Check if this was newly added to keep track of it
+		if new_element_id not in conflict:
+			if new_element_id not in duplicate:
+				added.append(new_element_id)
+
+		#FIXME: This exports the path and its overlap
+		if len(overlaps) > 0:
+			diff_xml = minidom.Document()
+			diff_svg = diff_xml.createElement("svg")
+			diff_svg.setAttribute("viewBox", viewBox)
+			diff_xml.appendChild(diff_svg)
+
+			append_svg_polygons(diff_xml, diff_svg, {new_element_id: new_polygon}, "fill:#008800;stroke:none")
+			for overlap_id, overlap_polygons in overlaps.items():
+				append_svg_polygons(diff_xml, diff_svg, {new_element_id + "-diff-%s-original" % overlap_id: old_polygons[overlap_id]}, "fill:#ff0000;stroke:none")
+				for i, overlap_polygon in enumerate(overlap_polygons):
+					append_svg_polygons(diff_xml, diff_svg, {new_element_id + "-diff-%s-overlap[%d]" % (overlap_id, i): overlap_polygon}, "fill:#ff8800;stroke:none")
+
+			open("tmp/%s.svg" % new_element_id, 'wb').write(diff_xml.toprettyxml('\t', '\n', 'UTF-8'))
+
+	return duplicate, touching, conflict, added
+
+def dump_polygons_to_svg(path, polygons):
+	xml = minidom.Document()
+	svg = xml.createElement("svg")
+	svg.setAttribute("width", width)
+	svg.setAttribute("height", height)
+	svg.setAttribute("viewBox", viewBox)
+	xml.appendChild(svg)
+
+	for polygon in polygons:
+		append_svg_polygons(xml, svg, polygon[0], polygon[1])
+
+	open(path, 'wb').write(xml.toprettyxml('\t', '\n', 'UTF-8'))
+
+
+def main(argv):
+	# Load and parse SVGs
+	old_xml = minidom.parse(argv[0])
+	old_polygons = get_svg_polygons(find_group(old_xml, "Top Traces"))
+	new_xml = minidom.parse(argv[1])
+	new_polygons = get_svg_polygons(find_group(new_xml, "Top Traces"))
+
+	# Re-export as polygons
+	dump_polygons_to_svg("tmp/old.svg", [(old_polygons, "fill:#000000;stroke:none")])
+	dump_polygons_to_svg("tmp/new.svg", [(new_polygons, "fill:#000000;stroke:none")])
+
+	duplicate, touching, conflict, added = compare(old_polygons, new_polygons)
+	_, _, _, removed = compare(new_polygons, old_polygons)
+
```

Comment:
- The `duplicate` should help with #7 to avoid exporting the same path twice.
- `touching` could be used to generate a netlist.
- The `conflict` should be reviewed by the user or reviewers, especially if the conflicting path is not `removed`.
- The `added` and `removed` should be used by reviewers.